release_notes.txt

```diff
@@ -80,14 +86,14 @@ system due to the number of changes in both PHP and MySQL. This means there may
 encountered in testing.
 
 If you are upgrading a older system and need help, please visit our support forums at
-http://xoops.org/modules/newbb/
+https://xoops.org/modules/newbb/
 
 How you can contribute to XOOPS
 -------------------------------
 Bug reports and feature requests: https://github.com/XOOPS/XoopsCore25/issues
 Patches and enhancements: https://github.com/XOOPS/XoopsCore25/blob/master/CONTRIBUTING.md
 Documentation: https://www.gitbook.com/book/xoops/xoops-documentation-process/details
-Support Forums: http://xoops.org/modules/newbb/
+Support Forums: https://xoops.org/modules/newbb/
 
 XOOPS Development Team
 2017
```

Comment:
Can we change it to 2019

Reply:
OK -- I was just trying to get back on schedule :smile: